training/src/anemoi/training/config/training/default.yaml

```diff
@@ -111,8 +111,6 @@ validation_metrics:
 
 variable_groups:
   default: sfc
-  pl:
```

Comment:
Instead of removing these lines entirely, I would maybe keep them as a comment with some text, just so that there is an example of what the user can run